apps/web-giddh/src/app/ledger/ledger.component.ts

```
@@ -879,12 +891,18 @@ export class LedgerComponent implements OnInit, OnDestroy {
                     this.reconcileClosingBalanceForBank.type = this.reconcileClosingBalanceForBank.type === 'CREDIT' ? this.localeData?.cr : this.localeData?.dr;
                 }
-
-                let checkedEntriesName: any[] = uniq([
-                    ...lt.debitTransactions?.filter(f => f.isChecked).map(dt => ({ uniqueName: dt.entryUniqueName, type: 'debit' })),
-                    ...lt.creditTransactions?.filter(f => f.isChecked).map(ct => ({ uniqueName: ct.entryUniqueName, type: 'credit' })),
-                ]);
-
+                let checkedEntriesName: any[];
+                if (this.ledgerView === LedgerViewEnum.TView) {
+                    checkedEntriesName = uniq([
+                        ...lt.debitTransactions?.filter(f => f.isChecked).map(dt => ({ uniqueName: dt.entryUniqueName, type: 'debit' })),
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)